src/Suave/project.json

```diff
@@ -78,9 +78,12 @@
             },
             "dependencies": {
                 "System.Data.Common": "4.0.1-rc2-23911",
+                "System.Diagnostics.Process": "4.1.0-rc-23911",
                 "System.Security.Cryptography.Primitives": "4.0.0-rc2-23911",
                 "System.IO.Compression": "4.1.0-rc2-23911",
                 "System.Net.Security": "4.0.0-rc2-23911",
+                "System.Globalization": "4.0.11-rc3-24008-00",
```

Comment:
no `-rc3`
the `System.Globalization` is already referenced by `NETStandard.Library`